<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prepWaterShellDialog</class>
 <widget class="QDialog" name="prepWaterShellDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Water Shell</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Streams</string>
   </property>
   <widget class="QComboBox" name="sCombo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>401</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Stream Source:</string>
    </property>
    <property name="buddy">
     <cstring>sCombo</cstring>
    </property>
   </widget>
   <widget class="QGroupBox" name="trajectoryRange">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>531</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Trajectory Range</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;End Index:</string>
     </property>
     <property name="buddy">
      <cstring>eIndex</cstring>
     </property>
    </widget>
    <widget class="QSpinBox" name="eIndex">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
     <property name="value">
      <number>9999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sIndex">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Start Index:</string>
     </property>
     <property name="buddy">
      <cstring>sIndex</cstring>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>571</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Output PDB File:</string>
    </property>
    <property name="buddy">
     <cstring>txtOutputPDB</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtOutputPDB">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOutputPDB">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>571</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Program Output</string>
   </property>
   <widget class="QPlainTextEdit" name="txtOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>551</width>
      <height>141</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>236</red>
          <green>236</green>
          <blue>236</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>236</red>
          <green>236</green>
          <blue>236</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnRun">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>sCombo</tabstop>
  <tabstop>trajectoryRange</tabstop>
  <tabstop>sIndex</tabstop>
  <tabstop>eIndex</tabstop>
  <tabstop>txtOutputPDB</tabstop>
  <tabstop>btnOutputPDB</tabstop>
  <tabstop>txtOutput</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
